### PR TITLE
zh-trans: add job.md

### PR DESCRIPTION
**Pull request by AdamDang**
zh-trans: add job.md

### CODE REVIEW

File: content/zh/docs/reference/glossary/job.md [hunk 1]
```diff
@@ -12,9 +28,18 @@ tags:
...
+<!--
  A finite or batch task that runs to completion.
+-->
+
+ Job 是需要运行完成的有限的或批量的任务。
 
 <!--more--> 
 
+<!--
 Creates one or more {{< glossary_tooltip term_id="pod" >}} objects and ensures that a specified number of them successfully terminate. As Pods successfully complete, the Job tracks the successful completions.
```
Comment (markthink) [inline content/zh/docs/reference/glossary/job.md]: the Job tracks the successful completions.

=> Job 作业将跟踪成功完成的情况。

File: content/zh/docs/reference/glossary/job.md [hunk 2]
```diff
@@ -12,9 +28,18 @@ tags:
 - core-object
 - workload
 ---
+-->
+
+<!--
  A finite or batch task that runs to completion.
+-->
+
+ Job 是需要运行完成的有限的或批量的任务。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/job.md]: finite task：确定性的任务？

File: content/zh/docs/reference/glossary/job.md [hunk 3]
```diff
@@ -12,9 +28,18 @@ tags:
...
 
 <!--more--> 
 
+<!--
 Creates one or more {{< glossary_tooltip term_id="pod" >}} objects and ensures that a specified number of them successfully terminate. As Pods successfully complete, the Job tracks the successful completions.
+-->
 
+Job 创建一个或多个 {{< glossary_tooltip term_id="Pod" >}} 对象，并确保指定数量的 Pod 成功终止。Pod 一旦成功完成，Job 就会跟踪成功完成的次数。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/job.md]: > as pods successfully complete

随着所创建的 Pods 成功结束，

File: content/zh/docs/reference/glossary/job.md (hunk 1 above)
Comment (AdamDang) [reply content/zh/docs/reference/glossary/job.md]: done！

File: content/zh/docs/reference/glossary/job.md (hunk 2 above)
Comment (AdamDang) [reply content/zh/docs/reference/glossary/job.md]: 很贴切，赞同！

File: content/zh/docs/reference/glossary/job.md (hunk 3 above)
Comment (AdamDang) [reply content/zh/docs/reference/glossary/job.md]: agree！

File: content/zh/docs/reference/glossary/job.md [hunk 4]
```diff
@@ -12,9 +28,18 @@ tags:
...
+
+ Job 是需要运行完成的确定性的或批量的任务。
 
 <!--more--> 
 
+<!--
 Creates one or more {{< glossary_tooltip term_id="pod" >}} objects and ensures that a specified number of them successfully terminate. As Pods successfully complete, the Job tracks the successful completions.
+-->
 
+Job 创建一个或多个 {{< glossary_tooltip term_id="Pod" >}} 对象，并确保指定数量的 Pod 成功终止。Pod 一旦成功完成，Job 作业就会随着所创建的 Pod 成功结束。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/job.md]:
```suggestion
Job 创建一个或多个 {{< glossary_tooltip term_id="Pod" >}} 对象，并确保指定数量的 Pod 成功终止。随着各 Pod 成功结束，Job 会跟踪记录成功完成的个数。
```

Comment (AdamDang) [reply content/zh/docs/reference/glossary/job.md]: done

**Pull request merged**